### PR TITLE
#884: Adds setUserStatus for LinkedIn

### LINKED ISSUE #884: 'setUserStatus' functionality is missed for LinkedIn in v3.0
## Bug, feature or question?

Is this a bug, a feature request or a question?

Give us a short description.

### Version and provider

What version of Hybrid Auth does this relates to?

Is this an issue with a provider? If yes, which one?

### Reproduction

How can we replicate this issue?

### PR DESCRIPTION
**Pull request by ApacheEx**
Resolves #884

### CODE REVIEW

File: docs/providers.md [hunk 1]
```diff
@@ -9,44 +9,44 @@ Supported providers
...
-BitBucket           | OAuth2  | [X]            | [X]            |                 |                |                      
-WordPress           | OAuth2  | [X]            | [X]            |                 |                |                      
-Tumblr              | OAuth1  | [X]            | [X]            | [X]             |                |                      
-Disqus              | OAuth2  | [X]            | [X]            |                 |                |                      
-Dribbble            | OAuth2  | [X]            | [X]            |                 |                |                      
-WindowsLive         | OAuth2  | [X]            | [X]            | [X]             |                |                      
-Foursquare          | OAuth2  | [X]            | [X]            | [X]             |                |                      
-Instagram           | OAuth2  | [X]            | [X]            |                 |                |                      
-LinkedIn            | OAuth2  | [X]            | [X]            |                 |                |
```
Comment (RajeshPerro) [inline docs/providers.md]: hi, I am new user. I have LnkedIn appid, Secret and AccessToken in my DB. How can I post a status by using setUserStatus() 
nb: though I have all those data so I don't want to push user in login again. Is it possible to post inLinkedIn by using this method.?

Comment (ApacheEx) [reply docs/providers.md]: Hi,
Check comment here: https://github.com/hybridauth/hybridauth/issues/888#issuecomment-354552714

Comment (RajeshPerro) [reply docs/providers.md]: Hi,
Thanks for your suggestion. But I am getting new issue. Please please help me!
Here is my method to post in LinkedIn but I am getting error : Fatal Error: 
ArrayObject Object
(
    [storage:ArrayObject:private] => Array
        (
            [exception] => Hybridauth\Exception\HttpRequestFailedException Object
                (
                    [message:protected] => Signed API request has returned an error. HTTP error 401. Raw Provider API response: {
  "errorCode": 0,
  "message": "Empty oauth2_access_token",
  "requestId": "GT3BPWBLSX",
  "status": 401,
  "timestamp": 1520772798106
}.
//This is my method........
 public function postToLinkedin($lnAppkey,$lnAppSec,$lnToken)
 {
         if($lnAppkey !=null && $lnAppSec !=null && $lnToken !=null)
        {
            $_SESSION['app_id_linkedIn'] = $lnAppkey;
            $_SESSION['app_sec_linkedIn'] = $lnAppSec;
            $_SESSION['access_token_linkedIn'] = $lnToken;
        }

        $config = [
            'callback' => $request->getBaseUrl().'/socialpost/application/postToLinkedin',
            "enabled" => true,
            'keys' => [
                'key' => $_SESSION['app_id_linkedIn'],
                'secret' => $_SESSION['app_sec_linkedIn']
            ],

        ];
// Instantiate LinkedIn Adapter.
        $linkedin = new Hybridauth\Provider\LinkedIn($config);

// Authenticate the user.
        $linkedin->setAccessToken($_SESSION['access_token_linkedIn']);

// Update the user status.
        $response = $linkedin->setUserStatus([
            'comment' => '',
            'content' => [
                'title' => 'This is a post',
                'description' => "I am trying to post a demo content",
                'submitted-url' => '',
                'submitted-image-url' => '',
            ],
            'visibility' => [
                'code' => 'anyone',
            ],
        ]);
        
 }

Comment (ApacheEx) [reply docs/providers.md]: Only one question, why here? why you add those comments in merged PR. Pls fill the issue if you have any troubles. 🤦‍♂️

**Pull request merged**